src/nilamP/StringOperations4.java

```diff
@@ -0,0 +1,51 @@
+/*Assignment - 25 : 20th Sep'2022
+
+Program 1: Print all the repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+output : Hindavi 
+         Shubham 
+
+Program 2: Print all the non repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham hindavi Shubham Titiksha Swati";
+output : Maulik 
+         Puja
+         Titiksha
+         Swati */
+package nilamP;
+
+public class StringOperations4 {
+	void getRepeatedNames(String str) {
+		String[] input = str.split(" ");
+		for (int i = 0; i < input.length; i++) {
+			boolean flag = true;
+			for (int j = i + 1; j < input.length; j++) {
+				if (input[i].equals(input[j]) && i != j) {
+					System.out.println(input[i]);
+				}
+			}
+		}
+	}
+
+	void getNonRepeatedNames(String str) {
```

Comment:
for(int i = 0;i<input.length;i++){
      if(str.indexOf(input[i]) == str.lastIndexOf(input[i])){
        System.out.println(input[i]);
      }
    }


you can use this code as well.. instead if two loops